x/bridge/keeper/assets.go

```diff
@@ -64,12 +61,18 @@ func (k Keeper) createAssets(ctx sdk.Context, assets []types.Asset) error {
 		// TODO: double-check if we need to handle the response
 		_, err := handler(ctx, msgCreateDenom)
 		if err != nil {
-			return errorsmod.Wrapf(
+			return nil, errorsmod.Wrapf(
 				types.ErrTokenfactory,
 				"Can't execute a create denom message for %s: %s", asset.Name(), err,
 			)
 		}
+
+		// TODO: set the last_transfer_height to the latest external blockchain height, since using 0
+		//  doesn't really make sense. Should use corresponding chain clients here after they are implemented.
+		asset.LastTransferHeight = 0
```

Comment:
I don't think we can use the corresponding chain clients here right? As in, it will have to be an input param provided by and agreed upon by the validator subset right?